docs/features/split_keyboard.md

```diff
@@ -139,6 +139,30 @@ Note that adding a diode at a previously unused intersection will effectively te
 
 While `MATRIX_MASKED` isn't necessary to use `SPLIT_HAND_MATRIX_GRID` successfully, without it you may experience issues trying to suspend your computer with your keyboard attached as the matrix will always report at least one key-press.
 
+
+For example, suppose you have the following keyboard.
+
+```c
+#define MATRIX_ROW_PINS { D4, C6, D7, E6, B4 }
+#define MATRIX_COL_PINS { F4, F5, F6, F7, B1, B3, B2 }
+
+/* COL2ROW, ROW2COL*/
+#define DIODE_DIRECTION COL2ROW
+```
+
+<img width="1389" src="https://i.imgur.com/MrcBQ78.png">
+
+
+You can add a diode to the circuit as below and further configuration as below.
+
+<img width="1389" src="https://i.imgur.com/GCi0WML.png">
+
+
+```c
+/* Split hand configration */
+#define SPLIT_HAND_MATRIX_GRID D4,B2
+```
+
```

Comment:
using `split.handedness.matrix_grid` may be preferable (eg json).  Especially as it will automatically add matrix masking config.